src/App.jsx

```diff
@@ -13,6 +14,7 @@ function App() {
                     <Route path="/" element={<LatestArticles/>}/>
                     <Route path="/latest" element={<LatestArticles/>}/>
                     <Route path="/topics/:topic_name" element={<TopicArticles/>}/>
```

Comment:
It's strange to me that you can click topic like a link and it takes you to an empty page while you choose from the dropdown. Is there a better way to implement the same functionality?